app/AppComponents.scala

```diff
@@ -30,12 +30,10 @@ class AppComponents(context: Context, identity: AppIdentity)
 
     override def controllerComponents: ControllerComponents = AppComponents.this.controllerComponents
 
-    override def panDomainSettings: PanDomainAuthSettingsRefresher = new PanDomainAuthSettingsRefresher(
+    override val panDomainSettings: PanDomainAuthSettingsRefresher = PanDomainAuthSettingsRefresher(
```

Comment:
The `def` → `val` is necessary due to:

* https://github.com/guardian/pan-domain-authentication/pull/155